### ISSUE TITLE
Selecting a desktop by name when name is not unique in multimonitor setup

### ISSUE THREAD

**Issue by DrPyser** [baskerville/bspwm#1073]
Currently, the desktop selector grammar does not support patterns like `[MONITOR_SEL:]desktop_name`, to select a desktop using its name and given the monitor. 

When desktop names are not unique, this means the name cannot be used to identify a desktop absolutely(except in the case of desktops on the local monitor using `$desktop_name.local`).

In my case, I use `btops` to automatically create desktops with specific names on each monitor. 
How would I choose a target desktop(e.g. for focusing or sending a window) on a specific monitor using the monitor name and desktop name?

Thanks!

**Comment by plitter** [comment]
You could find its ID and use that. `bspc query -D -m <monitor_id>` and from that find the correct desktop id.

**Comment by DrPyser** [comment]
@plitter yes, but how do I find the id if all I know is the name? See #1072.

**Comment by plitter** [comment]
@DrPyser Wouldn't `bspc query -D -d <desktop name>` give at least 2 ids in your case? About your other issue, I would try to verify your issue with `bspc query -T -m <monitor_id>` and see what it gives and `bspc query -D -m <monitor_id> --names`. Maybe also check that the desktop names through `bspc query -D -d <desktop_id> --names`.
